### DISCUSSION TITLE
Frontend Newbie: Project runs locally, builds successfuly, but not deploying. Infinite Loop

### DISCUSSION THREAD

**Discussion by SunnyWang0** [vercel/community#7355] [Help] [answered]
### Summary

I have a tailwind css project that is building correctly but does not deploy. It looks like it gets stuck after succesfully building. I have tried deploying through GitHub and vercel cli. The project contains a dist folder with assets and docs folders inside of the dict folder. Also inside the project folder is a src folder, and various config files. The project runs locally using npm install, npm run build and then using the VS Code Live Server extension. I am not familiar with frontend, what am I doing wrong?

<img width="281" alt="Screenshot 2024-07-02 at 11 45 38 PM" src="https://github.com/vercel/community/assets/71104760/7a91c77c-8a7d-4b95-91d7-74b8aa387d24">

^^ This is what my project directory looks like.

![image](https://github.com/vercel/community/assets/71104760/e5802c68-0adb-4242-b49a-a0c8a4729b76)
^^ This is all I see with the build terminal. No other error messages

project

### Steps to Reproduce

build this template project: https://github.com/SunnyWang0/templateWebsite

npm install
npm run build
and then view through VSCode Live Server Extension
Everything will run locally, however I try to deploy the project as is and it does not work

**Comment by pawlean** [comment, marked as answer]
Hi @SunnyWang0!

Could you try specifying in your "Root Directory" setting with `/dist` where your code lives?

![CleanShot 2024-07-03 at 13 23 01@2x](https://github.com/vercel/community/assets/17087373/48214ebf-4b29-4039-a7be-bed3b79c30d0)

See [relevant documentation. ](https://vercel.com/docs/deployments/configure-a-build) Let me know if I can help with anything else at all!

[a comment by an automated account is not shown]
